### PR TITLE
update to 1.7.1

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

**Comment by sannykr** [comment]
@synapticarbors : In this meta.yaml file, I used the hardcoded url to the pypi because the default one didn't work. Is it acceptable?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
+{% set version = "1.7.1" %}
+
 
 package:
     name: django-redis-cache
     version: {{ version }}
 
 source:
     fn: django-redis-cache-{{ version }}.tar.gz
-    url: https://pypi.python.org/packages/source/d/django-redis-cache/django-redis-cache-{{ version }}.tar.gz
```
Comment (isuruf) [inline recipe/meta.yaml:9]: @sannykr, this is outdated, see https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L18

Comment (sannykr) [reply recipe/meta.yaml:9]: @isuruf : I don't understand what you are saying. I had to update the package on conda-forge. When I changed the version number and the md5, I got an error about pypi url. That's why I have used the hardcoded url. I was just asking if it is acceptable to use the hardcoded url of the package.

Comment (ocefpaf) [reply recipe/meta.yaml:9]: @sannykr please read the example that @isuruf pasted above.

Comment (sannykr) [reply recipe/meta.yaml:9]: @isuruf : Thanks for letting me know. I have made the required changes.

**Comment by sannykr** [comment]
@kwilcox: Doesn't it work for windows?

File: recipe/meta.yaml (hunk 1 above)
Comment (sannykr) [reply recipe/meta.yaml:9]: @isuruf : Doesn't it work for windows? I don't know why it is disabled. 
When I looked at the pypi page, I found that it is OS independent. Should I use noarch: python in the build section?

**Comment by sannykr** [comment]
@isuruf: Could you please review and merge.

**Comment by isuruf** [comment]
I don't have permission. Please ping the maintainers

**Comment by kwilcox** [comment]
@sannykr As with the other PR, please rebase on top of master with (2) commits... the version bumpy and the re-render.

**Comment by sannykr** [comment]
@kwilcox : I have opened another pull request with only two commits. Please let me know if that works.

**Pull request closed without merging**